### ISSUE TITLE
AttributeError: 'ProjectResponse' object has no attribute 'id'

### ISSUE THREAD

**Issue by nikhase** [superannotateai/superannotate-python-sdk#623]
Hi together,

I am trying to retrieve custom item metadata by proejct_id and item_id.

I am using `superannotate==4.4.13`.

This is my code:
```python
from superannotate import SAClient

sa = SAClient(config_path="../superannotate.ini")
sa.get_item_by_id(project_id=12345 ,item_id=678910)  # here I use your IDs as int
```

While execution, I get the error ⬇️  for a response parsing.
Do you maybe have some changes in the backend?

Best, Niklas
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[13], line 1
----> 1 sa.get_item_by_id(project_id=139236 ,item_id=48526888)

File [~/.local/lib/python3.9/site-packages/superannotate/lib/app/interface/base_interface.py:239](https://vscode-remote+dev-002dcontainer-002b7b22686f737450617468223a222f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b222c22636f6e66696746696c65223a7b22246d6964223a312c22667350617468223a222f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b2f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c2265787465726e616c223a2266696c653a2f2f2f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b2f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c2270617468223a222f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b2f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c22736368656d65223a2266696c65227d7d.vscode-resource.vscode-cdn.net/workspaces/yolov8-fork/super_annotate/~/.local/lib/python3.9/site-packages/superannotate/lib/app/interface/base_interface.py:239), in Tracker.__call__(self, *args, **kwargs)
    237 except Exception as e:
    238     success = False
--> 239     raise e
    240 else:
    241     return result

File [~/.local/lib/python3.9/site-packages/superannotate/lib/app/interface/base_interface.py:236](https://vscode-remote+dev-002dcontainer-002b7b22686f737450617468223a222f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b222c22636f6e66696746696c65223a7b22246d6964223a312c22667350617468223a222f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b2f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c2265787465726e616c223a2266696c653a2f2f2f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b2f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c2270617468223a222f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b2f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c22736368656d65223a2266696c65227d7d.vscode-resource.vscode-cdn.net/workspaces/yolov8-fork/super_annotate/~/.local/lib/python3.9/site-packages/superannotate/lib/app/interface/base_interface.py:236), in Tracker.__call__(self, *args, **kwargs)
    234 success = True
    235 try:
--> 236     result = self.function(*args, **kwargs)
    237 except Exception as e:
    238     success = False

File [~/.local/lib/python3.9/site-packages/superannotate/lib/app/interface/types.py:51](https://vscode-remote+dev-002dcontainer-002b7b22686f737450617468223a222f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b222c22636f6e66696746696c65223a7b22246d6964223a312c22667350617468223a222f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b2f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c2265787465726e616c223a2266696c653a2f2f2f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b2f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c2270617468223a222f55736572732f6e696b6c61732f6465762f766f727765726b2f796f6c6f76382d666f726b2f2e646576636f6e7461696e65722f646576636f6e7461696e65722e6a736f6e222c22736368656d65223a2266696c65227d7d.vscode-resource.vscode-cdn.net/workspaces/yolov8-fork/super_annotate/~/.local/lib/python3.9/site-packages/superannotate/lib/app/interface/types.py:51), in validate_arguments..wrapped(self, *args, **kwargs)
     48 @wraps(func)
     49 def wrapped(self, *args, **kwargs):
     50     try:
---> 51         return pydantic_validate_arguments(func)(self, *args, **kwargs)
     52     except ValidationError as e:
...
     54         )
     55     except AppException as e:
     56         self._response.errors = e

AttributeError: 'ProjectResponse' object has no attribute 'id'
```

**Comment by VaghinakDev** [comment]
The issue will be fixed in the next release, thanks for opening the issue.

**Comment by VaghinakDev** [comment]
Fixed in version v4.4.14
